### PR TITLE
Allow `templateFilePath` to be set via config file

### PR DESCRIPTION
**Pull request by choumx**

### CODE REVIEW

**Comment by jeffposnick** [comment]
...and thanks for this, as well. 👍

**Comment by jeffposnick** [comment]
Sorry, could you clean up the [linting error](https://travis-ci.org/GoogleChrome/sw-precache/jobs/159973864#L845) prior to this getting merged?

```
[19:14:41] Starting 'lint'...
[19:14:41] 'lint' errored after 812 ms
[19:14:41] ESLintError in plugin 'gulp-eslint'
Message:
    Trailing spaces not allowed.
Details:
    fileName: cli.js
    lineNumber: 84
npm ERR! Test failed.  See above for more details.
```

File: cli.js [hunk 1]
```diff
@@ -81,6 +81,9 @@ function setDefaults(cli, configFileFlags) {
   compositeFlags.runtimeCaching = compositeFlags.runtimeCaching ||
     configFileFlags.runtimeCaching;
 
+  compositeFlags.templateFilePath = compositeFlags.templateFilePath ||
```
Comment (addyosmani) [inline cli.js]: LGTM once you've got the trailing space here addressed.

Comment (choumx) [inline cli.js]: Oops, done. Thanks!

**Comment by jeffposnick** [comment]
Thanks!

**Pull request merged**